.github/workflows/ci.yml

```diff
@@ -73,7 +74,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.12-dev']
+        python-version: ['3.13-dev']
```

Comment:
> Adds 3.13-dev test run. It currently fails, but will pass once trio and cffi push new releases.

I'd prefer to leave it at 3.12 for now so that all CI passes, and add a TODO pending the trio/cffi releases.

Reply:
I'll open an issue if trio/cffi have not released by the time #188 is finalized & merged